evm/p2p/discovery.py

```diff
@@ -111,21 +115,23 @@ def connection_made(self, transport):
         self.logger.debug("boostrapping with {}".format(self.bootstrap_nodes))
         await self.kademlia.bootstrap(self.bootstrap_nodes)
 
-    def datagram_received(self, data, addr):
+    # FIXME: Enable type checking here once we have a mypy version that
+    # includes the fix for https://github.com/python/typeshed/pull/17400
```

Comment:
This URL returns a 404.

Reply:
Oops, there was an extra trailing 0 there